doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -0,0 +1,13 @@
+API Changelog 
+=============
+
+.. contents:: |toctitle|
+    :local:
+    :depth: 1
+
+6.0.0
+-----
+
+Changes
+~~~~~~~
+    - **api/access/datafile**: When a null or invalid API Key is provided to download a public with this API call, it will result on a ``401`` error response. 
```

Comment:
.rst is tricky. The formatting is not right here. Please fix.

See https://dataverse-guide--10127.org.readthedocs.build/en/10127/api/changelog.html

Reply:
I checked the link, but I didn't notice anything weird or out of place. I can give you a call if you want to go through it or could you point me to what is wrong with it? 

Thanks!

Reply:
Fixed in 2433114ec7.

## before

<img width="844" alt="Screenshot 2023-11-17 at 11 19 38 AM" src="https://github.com/IQSS/dataverse/assets/21006/79f64f80-911e-4e25-8dec-068d58142275">

## after

<img width="872" alt="Screenshot 2023-11-17 at 11 19 54 AM" src="https://github.com/IQSS/dataverse/assets/21006/c13f70e1-c17c-4326-8e0c-62304ae0083b">

Reply:
Thanks!